packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -98,8 +99,9 @@ const _renderCards = (cards, containerRef) => (
                 src: ArrowRight20,
               },
             }}
-            target={card.cta.target}
             type="link"
+            role="region"
```

Comment:
Maybe this is just my accessibility ignorance showing, but I don't think we want to change the role of these. The role is a "link", and I'm not sure why we're wanting to change that. The `<a>` by default gets a a `Link` role. Based on my understanding we'd be hurting the accessibility of this if we change its role.

I'd personally love more information and reasoning from the QA team on why this is necessary.

Reply:
I have checked with @RaphaelAmadeu before making this change because he also got accessibility issue in `ContentGroupCards` it has `role=region`.